### DISCUSSION TITLE
Hide blog metadata and content alignment question

### DISCUSSION THREAD

**Discussion by mfridman** [squidfunk/mkdocs-material#4496] [Q&A] [answered]
I have 2 questions regarding the blog feature (apologies if I missed it in the docs or previous discussions).

1. Is it possible to hide the blog metadata (left-hand side, see image) similarly to `header.autohide`. That way a user reading the content can solely focus on the content? Also, ideally, this would be center-aligned irrespective of the left-hand side metadata.

<img src="https://user-images.githubusercontent.com/6278244/195747035-dd875b36-3b2f-45ca-9b37-cc690e682ea2.png" width="750" />

2. Some breakpoints don't align properly. I presume this is because of the (right) sidebar, is there a way to disable that and/or make it so the blog content is always center-aligned?

<img src="https://user-images.githubusercontent.com/6278244/195747333-e0c934ed-4e71-4c77-85cc-2bdb468b0f67.png" width="750" />

**Comment by squidfunk** [comment, marked as answer]
1. No, currently not, but you could adjust the `post.html` partial and remove the sidebar markup. Note that all of this is still a work in progress (the blog plugin is currently experimental) so it might change at any time, so it's not yet the time to introduce boolean flags for those features. We can revisit it when the dust settled.
2. I need a little more information what you mean by "don't align properly", as I don't quite understand what you mean. You can hide the table of contents as described in the documentation if you don't need it. Yes, it is also there when it is empty to keep page width consistent which is a deliberate decision, but I understand that some users don't like that which is why [it can be hidden](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#hiding-the-sidebars).

**Reply by mfridman** [reply]
This all sounds reasonable to me, thanks for your input. Closing.

If this isn't the best forum for blog questions please let me know.

**Reply by squidfunk** [reply]
Don't worry! You did everything right asking here 😊
